contract_partner/README.rst

```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+Contract Partner
+================
+
+This module was written to extend the functionality of contracts to support:
+* set contact and service partner in contract.
+* set contact and service partner from contract in invoice.
```

Comment:
The explanation for "contact" and "service" partner should be improved in the README:
* "contact" is the person or address where to send the invoices. If empty the "Customer" will be used. 
* "service" is where the service is rendered (site, location, address)

There is some overlapping with the [`service_desk`](https://github.com/OCA/project/tree/8.0/service_desk), addressing the same use cases but from the Project Task perspective:

The use case there is: 
1) I create a Task for a service
2) Pick a Contract (that has the `partner_id` "Customer" and an added `contact_id` Contact Partner)
3) Pick a Service Address. By default it's the Contact Partner but Customer child partners/addresses are selectable.

Looking at it, `contact_id` is in fact a mix between your contact and service fields. 
So I think that it should be deprecated to use the fields added here.